### PR TITLE
Pablo/docstring, tags, and extended a step

### PR DESCRIPTION
**Pull request by mefemefe**
-Added doc strings for all my steps, and extended the functionality of:
`'I receive a response with the "{item}" "{attribute}"'`
-Added more tests and tags for 'smoke' and 'negative'

### CODE REVIEW

**Review by nicolasvallejo13 — APPROVED** [review]

**Review by trab2 — APPROVED** [review]

File: tests/api/features/attachments.feature [hunk 1]
```diff
@@ -18,6 +19,14 @@ Feature: Trello API Attachments
         Then I receive a response with the "attachment" schema
         And the status code is "200"
 
+    @negative
+    Scenario: GET an non existing attachment from a card
+        Given I created a new card
+        And I created an attachment on the card
+        When I send a "GET" request to "/cards/{card}/attachments/X"
+        Then the response contains an error message
```
Comment (atbootcamp) [inline tests/api/features/attachments.feature:27]:
```suggestion
        Then the response contains the error message "invalid"
```

**Review by atbootcamp — CHANGES_REQUESTED** [review]
